internal/output/html.go

```diff
@@ -103,6 +130,8 @@ var baseImages = []string{"Debian", "Alpine", "Ubuntu"}
 //go:embed html/*
 var templates embed.FS
 
+var isContainerScanning = false
```

Comment:
Prefer not to have a global variable here (especially since if this code is ran twice this will break, which will be hard to debug).

Reply:
We can have a separate function that does it's own loop and figure out whether there are image origin fields.